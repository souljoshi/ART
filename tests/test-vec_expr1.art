void printfln(double x)
{
    printf(x);printc('\n');
}
int main()
{
	vec v1;
	vec v2;
	double f1;
	double f2;

	f1 = 2.0;
	f2 = 5.5;

    v1 = <1.0 + 1.0, 55.1 + 2.0>;

    v2 = <f1 * f2, f2 / 2>;

    printfln(v1[0]);
    printfln(v1[1]);
    printfln(v2[0]);
    printfln(v2[1]);

}