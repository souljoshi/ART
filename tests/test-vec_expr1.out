2.000000
57.100000
11.000000
2.750000
